octopusdeploy_framework/schemas/schema.go

```diff
@@ -145,6 +147,9 @@ func GetIdResourceSchema() resourceSchema.Attribute {
 		Description: "The unique ID for this resource.",
 		Computed:    true,
 		Optional:    true,
+		PlanModifiers: []planmodifier.String{
```

Comment:
These changes match the util.schema class. It'd be nice to fully consolidate these two files now that things have settled.